tests/epyccel/test_epyccel_return_arrays.py

```diff
@@ -0,0 +1,148 @@
+# pylint: disable=wrong-import-position, disable=missing-function-docstring, missing-module-docstring/
```

Comment:
Why do you disable `wrong-import-position`? Please don't disable pylint errors without good reason

`reimported` has a good reason. You can disable this